File: KyGreenPet/app/src/main/java/cc/bitky/kygreenpet/bean/KyOperation.java
package cc.bitky.kygreenpet.bean;

import cn.bmob.v3.BmobObject;
import cn.bmob.v3.datatype.BmobDate;

public class KyOperation extends BmobObject {

  private Double tempValue;
  private Boolean isHumidity;
  private Boolean isOpened;
  private Boolean isAuto;
  private BmobDate dateTime;

  public Double getTempValue() {
    return tempValue;
  }

  public void setTempValue(Double tempValue) {
    this.tempValue = tempValue;
  }

  public Boolean getHumidity() {
    return isHumidity;
  }

  public void setHumidity(Boolean humidity) {
    isHumidity = humidity;
  }

  public Boolean getOpened() {
    return isOpened;
  }

  public void setOpened(Boolean opened) {
    isOpened = opened;
  }

  public Boolean getAuto() {
    return isAuto;
  }

  public void setAuto(Boolean auto) {
    isAuto = auto;
  }

  public BmobDate getDateTime() {
    return dateTime;
  }

  public void setDateTime(BmobDate dateTime) {
    this.dateTime = dateTime;
  }
}

File: KyGreenPetWPF/KyGreenPet/bean/KyOperation.cs
using System;
using cn.bmob.io;

namespace KyGreenPet.bean
{
    public class KyOperation : BmobTable
    {
        public BmobDouble tempValue { set; get; }
        public BmobBoolean isHumidity { set; get; }
        public BmobBoolean isOpened { set; get; }
        public BmobBoolean isAuto { set; get; }
        public BmobDate dateTime { set; get; }

        public KyOperation()
        {

        }

        public KyOperation(double tempValue, bool isHumidity)
        {
            this.tempValue = tempValue;
            this.isHumidity = isHumidity;
            isOpened = true;
            dateTime = DateTime.Now;
        }


        //读字段信息
        public override void readFields(BmobInput input)
        {
            base.readFields(input);
            tempValue = input.getDouble("tempValue");
            isHumidity = input.getBoolean("isHumidity");
            isOpened = input.getBoolean("isOpened");
            isAuto = input.getBoolean("isAuto");
            dateTime = input.getDate("dateTime");
        }

        //写字段信息
        public override void write(BmobOutput output, bool all)
        {
            base.write(output, all);

            output.Put("tempValue", tempValue);
            output.Put("isHumidity", isHumidity);
            output.Put("isOpened", isOpened);
            output.Put("isAuto", isAuto);
            output.Put("dateTime", dateTime);
        }
    }
}